lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -3447,6 +3506,7 @@ class DecomposeComplexOpsPass
         DecomposeConstantTensorAllocLikeOp<AtenOnesLikeOp, 1>>(patterns);
     addPatternIfTargetOpIsIllegal<
         DecomposeConstantTensorAllocLikeOp<AtenZerosLikeOp, 0>>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenStackOp>(patterns);
```

Comment:
You also need to add a similar line here:

https://github.com/llvm/torch-mlir/blob/86b792520bb1331f106f7eae2f38fb18db70c397/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L335

Reply:
Done!